intermediate_source/char_rnn_generation_tutorial.py

```diff
@@ -5,7 +5,7 @@
 **Author**: `Sean Robertson <https://github.com/spro>`_
 
 This is our second of three tutorials on "NLP From Scratch".
-In the `first tutorial </intermediate/char_rnn_classification_tutorial>`
+In the first tutorial :doc:`/intermediate/char_rnn_classification_tutorial`
```

Comment:
You can just make a regular url. There is an underscore missing at the end:
```suggestion
In the first tutorial `/intermediate/char_rnn_classification_tutorial`_
```